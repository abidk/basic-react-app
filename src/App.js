import React from 'react';
import List from './components/organisms/List';
import Form from './components/organisms/Form';
import Names from './service/RetrieveNames';
import './App.css';

function App() {
  return (
    <div className="App">
      <h1 className="App-title">Basic React App</h1>
      <div className="App-form">
        <Form />
      </div>
      <div className="App-items">
        <List names={Names} />
      </div>
    </div>
  );
}

export default App;
