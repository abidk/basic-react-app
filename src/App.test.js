import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import App from './App';

describe('Test main app', function () {

  it('should contain list', () => {
    const output = shallow(<App />);

    expect(shallowToJson(output)).toMatchSnapshot();
  });
});