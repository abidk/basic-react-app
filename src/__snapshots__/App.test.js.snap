// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test main app should contain list 1`] = `
<div
  className="App"
>
  <h1
    className="App-title"
  >
    Basic React App
  </h1>
  <div
    className="App-form"
  >
    <ConnectFunction />
  </div>
  <div
    className="App-items"
  >
    <ConnectFunction
      names={
        Array [
          "Joe",
          "Abid",
        ]
      }
    />
  </div>
</div>
`;
